slasher/db/kv/kv.go

```diff
@@ -31,12 +33,20 @@ type Config struct {
 
 // Close closes the underlying boltdb database.
 func (db *Store) Close() error {
+	db.spanCache.Purge()
 	return db.db.Close()
 }
 
+// RemoveOldestFromCache clears the oldest key out of the cache obly if the cache is at max capacity.
+func (db *Store) RemoveOldestFromCache(ctx context.Context) uint64 {
+	ctx, span := trace.StartSpan(ctx, "slasherDB.removeOldestFromCache")
+	defer span.End()
```

Comment:
This span doesn't do anything, as it will run before the return, see https://play.golang.org/p/m-wXudF_BT-

Reply:
I see! Thank you for the tip, I'll fix it.